scripts/_common.sh

```diff
@@ -79,6 +87,7 @@ if [ -z "$RID" ]; then
     fi
 fi
 
+export REPOROOT=$(cd $COMMONDIR/.. && pwd)
```

Comment:
The DIR variable was overwriting the DIR in whatever script it got sourced in. Had me pretty confused for a bit too.